src/Raven.Studio/typescript/viewmodels/manage/configureMicrosoftLogsDialog.ts

```diff
@@ -0,0 +1,50 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import dialog = require("plugins/dialog");
+import aceEditorBindingHandler = require("../../common/bindingHelpers/aceEditorBindingHandler");
+
+interface ConfigureMicrosoftLogsDialogResult {
+    isEnabled: boolean;
+    configuration: string;
+}
+
+class configureMicrosoftLogsDialog extends dialogViewModelBase {
+    
+    code = ko.observable<string>("");
+    isEnabled = ko.observable<boolean>(false);
+    
+    constructor(isEnabled:boolean, configuration: string) {
+        super();
+
+        aceEditorBindingHandler.install();
+
+        this.code(configuration);
+        this.isEnabled(isEnabled);
+    }
+
+    attached() {
```

Comment:
remove?

Reply:
I want to disable this method, so that pressing enter does not call save button click

Reply:
so either add comment why method body is empty or override method protected enterKeyPressed(): boolean  and put comment in body that we want to disable this behavior